client/src/components/UserRegisterForm.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import { Button, Form, Grid } from "semantic-ui-react";
+
+export class UserRegisterForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      username: "",
+      password: "",
+      confirmationpassword: ""
+    };
+  }
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+  handleSubmit = e => {
+    e.preventDefault();
+
+    const UserRequest = {
+      method: "POST",
+      body: JSON.stringify({
+        email: this.state.email,
+        username: this.state.username,
+        password: this.state.password,
+        confirmationpassword: this.state.confirmationpassword
+      }),
+      headers: {
+        "Content-Type": "application/json"
+      }
+    };
+    fetch("http://localhost:4000/auth/register", UserRequest)
+      .then(res => res.json())
+      .then(data => {
+        console.log(data);
```

Comment:
Remove the console.log, this is usually just used for debugging but should not be left in the code after.

Reply:
yes its removed